spec/nylas/model_spec.rb

```diff
@@ -40,6 +40,10 @@ def example_instance_hash
       expect(instance.api).to eql(api)
     end
 
+    it "instantiates gracefully even if the api responds with additional fields" do
```

Comment:
Is this the only place tests should be updated for this change?

Reply:
I'm not sure which places would be good to add this? This is testing it as close to the base implementation as possible, which means it's less likely to fail for weird reasons.

Reply:
Well, the other place this *could* bite us is when someone using `update` on a model; which is only called by users of the library. I think in those cases it's probably good to raise an error, as we want to make sure people aren't trying to update attributes that don't exist. That said a more intent-revealing error could be more appropriate.

Reply:
@HayleyCAnderson - I've changed update to raise an exception if a user attempts to update a field that is not part of the API schema supported by the gem.